examples/deferred_processing/README.md

```diff
@@ -0,0 +1,340 @@
+# Deferred Processing and Transactions
+
+## Introduction
```

Comment:
Let's get rid of the "Introduction" title. The *Deferred Processing and Transactions* section is empty. And we want to avoid that. Alternatively, you could add a paragraph under it. But, IMO, just kill the Introduction title.